oneEyedSunday/sendIT
# #161977733 Utilize Persistence with a DB

#### What does this PR do?
Utilize the provisioned database for persistence 
#### Description of Task to be completed?
* Utilize the database provisioned for persisitence
#### How should this be manually tested?
* Install postgres
* create an .env file in the server directory
* follow the example given in the .example.env file to provide credentials
* Setup script will run after build and create needed tables
* Use Postman to make a GET request to endpoints to check.
* Open up your postgres database terminal and check.
* Check the [hosted version here](ispoa-sendit.herokuapp.com/api) and test with POSTMAN also

#### Context
Test script should fail as I've not updated them to reflect uuid as id field.
Would update tests.

#### What are the relevant pivotal tracker stories?
#161977733
